contracts/libraries/Message.sol

```diff
@@ -98,6 +97,7 @@ library Message {
         bytes32[] _ss,
         IBridgeValidators _validatorContract) internal view {
         require(isMessageValid(_message));
+        require(_vs.length == _rs.length && _rs.length == _ss.length);
```

Comment:
Do you think it is really needed? If validator is not misbehaving it will always true (spend extra gas). Whereas if the validator did something incorrectly the transaction will be reverted in the line 107 anyway.

Reply:
I think you're right. The validator won't be able to misbehave anyway, so this will only hurt with extra gas on valid transactions. I think we can close this PR without merging.

Reply:
This is good that you prepared tests to check this functionality - it makes sense to have them. So, we will close the PR but rollback the changes in the discussed line of the code.

Probably it will be good idea also to add a comment prior to this line which says something like 'It is not necessary to check that arrays have the same length since it will be handled during attempt to access to the corresponding elements in the loop and the call will be reverted. It will save gas for the rational validators actions and still be be safe enough from security point of view'

Reply:
Thanks! Removed the check and added a comment on the method https://github.com/poanetwork/poa-bridge-contracts/pull/204/commits/151288af8c5d59afbc092e79155a5a3ffbdf43f0